File: app/dbmodels/LinkedIpNetwork.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'LinkedIpNetwork',
    'menu'          => ''
);

$table['fields'] = array(
    'id'               => DBModels::type('increments', 
                                         array('visible' => false)),
    'ip_address_id'     => DBModels::type('integer'),
    'ip_network_id'    => DBModels::type('integer'),
);

File: fusionglpi/database/linked_ipnetwork.py
from fusionglpi import db


class LinkedIpnetwork(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    ip_address_id = db.Column(db.Integer, nullable=False, server_default="0")
    ip_network_id = db.Column(db.Integer, nullable=False, server_default="0")
